code/src/main/kotlin/com/expediagroup/sdk/v2/core/client/util/ApiClientHttpEngineUtil.kt

```diff
@@ -0,0 +1,21 @@
+package com.expediagroup.sdk.v2.core.client.util
+
+import com.expediagroup.sdk.v2.core.client.ApiClientApolloHttpEngine
+import com.expediagroup.sdk.v2.core.trait.configuration.ClientConfiguration
+
+/**
+ * Creates an instance of `ApiClientApolloHttpEngine` using the provided namespace and client configuration.
+ *
+ * @param namespace The namespace to be used for creating the API client.
+ * @param configuration The client configuration implementing `ClientConfiguration` interface.
+ * @return An instance of `ApiClientApolloHttpEngine`.
+ */
+fun createApiClientHttpEngine(
```

Comment:
I think the name of this function can be a misleading. The name is pretty generic but it's actually returning something specific to Apollo custom engine adaptation. Maybe it's better to remove this function and explicitly use `ApiClientApolloHttpEngine` where needed

Reply:
Done.